src/server/qgsserverparameters.cpp

```diff
@@ -200,6 +206,78 @@ QgsRectangle QgsServerParameterDefinition::toRectangle( bool &ok ) const
   return extent;
 }
 
+QString QgsServerParameterDefinition::loadUrl( bool &ok ) const
+{
+  ok = true;
+
+  // Get URL
+  QUrl url = toUrl( ok );
+  if ( !ok )
+  {
+    return QString();
+  }
+
+  // fetching content
+  QgsNetworkContentFetcher fetcher;
+  QEventLoop loop;
+  QObject::connect( &fetcher, &QgsNetworkContentFetcher::finished, &loop, &QEventLoop::quit );
+
+  QgsMessageLog::logMessage( QStringLiteral( "Request started [url: %1]" ).arg( url.toString() ) );
```

Comment:
tr(...)

Reply:
Just wondering what happens here if the connection requires authentication ... unfortunately the current implementation of authmanager cannot automatically inject the authentication configuration based on a domain (and we probably don't want to do that without user confirmation or authentication configuration).

Not a blocker, but something that needs to be considered.

Reply:
@pblottiere which class in the server is provided tr(...) ?

Reply:
found ad done